KMaina/Fast-Food-App
# #160648386 User Order Page redesign

#### What does this PR do?
Allows a user to order for a food item.
#### Description of Task to be completed?
A user should indicate the quantity of food items they need then add to their cart
#### How should this be manually tested?

- Clone the repo onto your local machine.

- Open a terminal and run cd Fast-Food-App

- run `git checkout -b ch-redesing-user-order-160648386`

- run `git pull https://github.com/KMaina/Fast-Food-App.git ch-redesing-user-order-160648386`

- In the folder where you cloned the repo, you should open the UI folder.

- Open the `user-order.html` to see it in your browser for review.
#### Any background context you want to provide?
This is a revision of the user-order.html page. It was done to make improvements to the UI.
#### What are the relevant pivotal tracker stories?
#160648386
